leaders.tex

```diff
@@ -6,6 +6,7 @@ \section{LEADERS}
 
 Leaders aid in melee combat and rally troops. Saxon leaders determine the disposition of Saxon wings. Leaders can be killed, wounded, or captured, affecting victory conditions at the end of the game.
 
+\addtocontents{toc}{\addtolength{\cftsubsecnumwidth}{0.4em}}
```

Comment:
This forces some horizontal spacing between the subsection number and the text. This was necessary because once we went to 10+, the extra numbers would get crunched up.